docs/_interface/Private-Network.md

```diff
@@ -0,0 +1,291 @@
+adjusted automatically so that new blocks are created approximately 12 seconds apart. As
+more mining resources are deployed on the network, creating a new block becomes harder so
+that the average block time matches the target block time.
+
+Clique consensus is a proof-of-authority system where new blocks can be created by
+authorized 'signers' only. The clique consenus protocol is specified in
+[EIP-225](clique-eip). The initial set of authorized signers is configured in the genesis
+block. Signers can be authorized and de-authorized using a voting mechanism, thus allowing
+the set of signers to change while the blockchain operates. Clique can be configured to
+target any block time (within reasonable limits) since it isn't tied to the difficulty
+adjustment.
+
+[clique-eip]: https://eips.ethereum.org/EIPS/eip-225
+
+### Creating The Genesis Block
+
+Every blockchain starts with the genesis block. When you run Geth with default settings
+for the first time, it commits the main net genesis to the database. For a private
+network, you usually want a different genesis block.
+
+The genesis block is configured using the _genesis.json_ file. When creating a genesis
+block, you need to decide on a few initial parameters for your blockchain:
+
+- Ethereum platform features enabled at launch (`config`). Enabling protocol features
+  while the blockchain is running requires scheduling a hard fork.
+- Initial block gas limit (`gasLimit`). Your choice here impacts how much EVM computation
+  can happen within a single block. We recommend using the main Ethereum network as a
+  guideline to find a good amount. The block gas limit can be adjusted after launch using
+  the `--targetgaslimit` command-line flag.
+- Initial allocation of ether (`alloc`). This determines how much ether is available to
+  the addresses you list in the genesis block. Additional ether can be created through
+  mining as the chain progresses.
+
+#### Clique Example
+
+This is an example of a genesis.json file for a proof-of-authority network. The `config`
+section ensures that all known protocol changes are available and configures the 'clique'
+engine to be used for consensus.
+
+Note that the initial signer set must be configured through the `extradata` field. This
+field is required for clique to work.
+
+First create the signer account keys using the [geth account](./managing-your-accounts)
+command (run this command multiple times to create more than one signer key).
+
+```shell
+geth account new --datadir data
+```
+
+Take note of the Ethereum address printed by this command.
+
+To create the initial extradata for your network, collect the signer addresses and
+hex-encode `extradata` as the concatenation of 32 zero bytes, all signer addresses, and 65
+further zero bytes. In the example below, `extradata` contains a single initial signer
+address, `0x7df9a875a174b3bc565e6424a0050ebc1b2d1d82`.
+
```

Comment:
`extradata = "0x" + (32 times "0") + bareAddress1 + bareAddress2 + ... + (65 times "0")`
where bare addresses are the addresses without the "0x" prefix.